lib/config/presets/internal/helpers.ts

```diff
@@ -21,6 +21,23 @@ export const presets: Record<string, Preset> = {
     description: 'Keep `typescript` version in sync with the `rc` tag.',
     extends: [':followTag(typescript, rc)'],
   },
+  goXPackagesChangelogLink: {
+    description: 'Correctly link to the source code for golang.org/x packages',
+    matchManagers: ['gomod'],
+    matchUpdateTypes: ['major', 'minor', 'patch', 'digest'],
```

Comment:
I've added `digest` here, which wasn't present in yours on Forgejo - any reason?

Reply:
doesn't work for digest updates